ooniprobe/View/ResultViewController.m

```diff
@@ -60,21 +70,34 @@ -(void) loadScreen :(NSString*) content{
 }
 
 -(void) selectTest{
+    [self performSegueWithIdentifier:@"toResultSelector" sender:self];
+    /*
     UIActionSheet *actionSheet = [[UIActionSheet alloc]
                                   initWithTitle:NSLocalizedString(@"select_test", nil)
                                   delegate:self
                                   cancelButtonTitle:nil
                                   destructiveButtonTitle:nil
                                   otherButtonTitles:nil];
-    NSArray *tests = [self getItems];
-    for(NSString *content in tests){
+    for(NSString *content in items){
         NSData *data = [content dataUsingEncoding:NSUTF8StringEncoding];
         NSDictionary *json = [NSJSONSerialization JSONObjectWithData:data options:0 error:nil];
         [actionSheet addButtonWithTitle:[NSString stringWithFormat:@"input %@", [json objectForKey:@"input"]]];
     }
     actionSheet.actionSheetStyle = UIActionSheetStyleDefault;
     actionSheet.tag = 1;
     [actionSheet showInView:self.view];
+     */
```

Comment:
Why is this code commented out? Can't we remove it, instead, if it's not necessary?